Codeinwp/neve
# Change product buy actions css

### Summary
Did some changes here in order to achieve this https://github.com/Codeinwp/neve-pro-addon/pull/1328

### Will affect visual aspect of the product
YES

### Test instructions
- Test with different types of products: Grouped, External or Variable
- Make sure you test this without Neve Pro and see if everything works ok.
- You should check the single product page. I've made a change so that the input and the buy button would have the same height. If the Button's padding is set as 100px, the quantity input should have the same height.
